src/main/java/org/sagebionetworks/bridge/config/Config.java

```diff
@@ -0,0 +1,30 @@
+package org.sagebionetworks.bridge.config;
+
+import java.util.List;
+
+public interface Config {
```

Comment:
Trying to extract a basic API by inserting an interface.  Even though it has only one implementation, the interface used correctly can insulate changes.